admin/python/lsst/qserv/admin/qservAdmin.py

```diff
@@ -561,9 +576,36 @@ def addNode(self, nodeName, nodeType, host, runDir, mysqlConn):
 
         # make a node and set all options
         options = dict(type=nodeType, host=host, runDir=runDir, mysqlConn=mysqlConn)
-        self._kvI.create(nodeKey, "")
+        self._kvI.create(nodeKey, str(state))
         self._addPacked(nodeKey, options)
 
+    def setNodeState(self, nodes, state):
+        """
+        Change the state of the node (or multiple nodes).
+
+        @param nodes:    Lsit of node names or single node name
+        """
+
+        if state not in NodeState.STATES:
+            raise QservAdminException(QservAdminException.WRONG_PARAM_VAL, 'state: ' + str(state))
+
+        # first check version
+        self._versionCheck()
+
+        # if argument is a string then it's a single node name
+        if isinstance(nodes, types.StringTypes):
+            nodes = [nodes]
+
+        for node in nodes:
+            # check that node exists
+            nodeKey = '/NODES/' + node
+            if not self._kvI.exists(nodeKey):
+                raise QservAdminException(QservAdminException.NODE_DOES_NOT_EXIST,
+                                          'Node: ' + node, 'Key: ' + nodeKey)
+
+            # update its state
```

Comment:
See my other comment. And I think kazoo will print a lot of info here if you enable logging for kazoo.